### PR TITLE
Add external links

### LINKED ISSUE #235: Add link to Terms of Service
We should have a prominent link to the Elastic Maps Terms of Service on the landing page.

### PR DESCRIPTION
**Pull request by nickpeihl**
Fixes #235 and #140 

This PR would add links to the EMS Terms of Service and this GitHub repo to https://maps.elastic.co. 

Additionally, the link for reporting data issues is changed to the ems-file-service repo.

cc @mbarretta @richardim

### CODE REVIEW

File: public/js/components/app.js [hunk 1]
```diff
@@ -219,9 +221,31 @@ export class App extends Component {
                 </EuiPageContentBody>
               </EuiPageContent>
               <EuiSpacer />
-              <EuiText size="xs" textAlign="center">
-                <p>Please submit any issues with this layer or suggestions for improving this layer in the <a href="https://github.com/elastic/kibana/issues/new" target="_blank">Kibana repo</a>.</p>
-              </EuiText>
+              <EuiFlexGroup justifyContent="spaceAround">
+                <EuiFlexItem grow={false}>
+                  <p>
```
Comment (thomasneirynck) [inline public/js/components/app.js:226]: nice fix of no longer pointing to kibana repo

**Review by thomasneirynck — APPROVED** [review]
code review lgtm

**Review by jsanz — CHANGES_REQUESTED** [review]
What about having the icons in the top navigation bar?

image


I had to update EUI to the last version, but this is the snippet of the header

```xml
<EuiHeader>
  <EuiHeaderSectionItem border="right">
    <EuiHeaderLogo href="#" aria-label="Go to elastic.co" iconType="emsApp" >Elastic Maps Service</EuiHeaderLogo>
  </EuiHeaderSectionItem>
  <EuiHeaderSectionItem border="none">
    <EuiHeaderLinks>
      <EuiHeaderLink href="https://elastic.co">elastic.co</EuiHeaderLink>
      <EuiHeaderLink href="https://www.github.com/elastic/ems-landing-page"> Github </EuiHeaderLink>
      <EuiHeaderLink href="https://www.github.com/elastic/ems-file-service/issues/new"> Report data issues </EuiHeaderLink>
      <EuiHeaderLink href="https://www.elastic.co/elastic-maps-service-terms"> Terms of Service </EuiHeaderLink>
    </EuiHeaderLinks>
  </EuiHeaderSectionItem>
</EuiHeader>
```

**Comment by nickpeihl** [comment]
> What about having the icons in the top navigation bar?
> 
> image

Good idea, however, I was thinking of reserving space on the top bar for identifying the EMS version (https://github.com/elastic/ems-landing-page/issues/203) and possibly changing the current EMS version (select box?) if we implement client-side routing (https://github.com/elastic/ems-landing-page/issues/226). Do you think we have enough space there for all of that? Or maybe the EMS version goes somewhere else?

**Comment by jsanz** [comment]
I like not having to scroll down for those links, we can shorten them a bit like

`Version 7.10 | Code | Issues | Terms`


I don't think there will be space issues, and I like EUI frontpage header.

**Review by jsanz — APPROVED** [review]
:+1:

**Pull request merged**